akirachix/Magezi-Informational
# Add footer feature

# Engineers Checklist

This PR implements a footer component for the Shawazi informational website which provides additional navigation and information.

## What this PR does?

* What is this?
This PR adds a new footer component to the existing layout of the application.

* Why this?
This footer component provides the necessary navigation tools and necessary information to help the user.

* How are you doing this? 

- Create a new footer component in the components directory.
- Integrate the footer into the main layout (page.tsx).
- Add the necessary styling.

*Updates for features, enhancements, or bugfixes

- Added responsiveness to the footer component.
- Added social media links.

## Type of change

Specify the kind of PR/MR

Please delete options that are not relevant.


## How can it be tested?

- Pull the latest changes from the feature/footer branch
- Run the application using npm run dev
- Follow the link and observe how it behaves.
- Check all links and ensure it is responsive.

# Where is it documented?
[- List JIRA tickets](https://nakatozainamusana.atlassian.net/jira/software/projects/SCRUM/boards/4/backlog?selectedIssue=SCRUM-52)
